### ISSUE TITLE
Accessibility:  new placeholder.png doesn't pass contrast check

### ISSUE THREAD

**Issue by mikeyarce** [woocommerce/woocommerce#17936]
The new placeholder can have some contrast issues because of it's transparency.

On a white background, it will lack enouch contrast, on a dark background it would be ok, but on a grey background it would be invisible:

On Grey:
![screen shot 2017-11-28 at 9 01 51 am](https://user-images.githubusercontent.com/3220162/33333075-cab53d2e-d41a-11e7-90ed-7bfe6faa2039.png)

On White:
![screen shot 2017-11-28 at 9 02 12 am](https://user-images.githubusercontent.com/3220162/33333093-d5fb4e44-d41a-11e7-97c2-8f3d000dbbf1.png)

On Black:
![screen shot 2017-11-28 at 9 03 15 am](https://user-images.githubusercontent.com/3220162/33333138-fbd0f966-d41a-11e7-931d-07078556baf9.png)

Maybe what we could do is add a white background to the placeholder image, and make the grey darker so that no matter what the background of the theme, the contrast would be sufficient.
A good accessibility contrast checker can be found here: https://webaim.org/resources/contrastchecker/
cc @kellychoffman 

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [x] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [x] I have attempted to find the simplest possible steps to reproduce the issue
- [ ] I have included a failing test as a pull request (Optional)

**Comment by kellychoffman** [comment]
+1 for adding white to the background.

**Comment by mikejolley** [comment]
Pardon my ignorance, but why does the contrast of a placeholder/watermark matter? It has no visual meaning really and is just there as a default. If this were the contrast of text I'd understand the issue.

> WCAG 2.0 level AA requires a contrast ratio of 4.5:1 for normal text and 3:1 for large text. Level AAA requires a contrast ratio of 7:1 for normal text and 4.5:1 for large text.

This is not text.

**Comment by kellychoffman** [comment]
Sure, but let's still add a white background so you can actually see the image.

**Comment by claudiosanches** [comment]
I prefer white background too, even not looking for accessibility issues, that transparent bg looks terrible :P
